todo_app/app.py

```diff
@@ -16,19 +17,58 @@
 def index():
 
     cardList = get_items()
-    
-        
+     
     return render_template('index.html', cardList=cardList)    
     
 
 @app.route('/add', methods = ['POST'])
 def add_Task():
     
-    # get the title data from the Add Task form field in index.html
-    # add it to the list
-    newTask = request.form.get('newTask')
-    add_item(newTask)
+    title = request.form.get('title')
+    description = request.form.get('description')
+    list = request.form.get('listName')
+    listId = get_list_by_name(list)
+    
+    add_item(title, description, listId) 
     
-    # send user back to starting app route
     return redirect('/')
-    
\ No newline at end of file
+
+
+@app.route('/task/', methods =['GET'])
+def get_Task():
+
+    taskId = request.args.get('taskId')
+    taskName = request.args.get('taskName')
+    taskDescription = request.args.get('taskDescription')
+    taskListName =  request.args.get('taskListName')
+
+    return render_template('task.html', taskId=taskId, taskName=taskName, taskListName=taskListName, taskDescription=taskDescription)
+
+
+@app.route('/update/', methods=['GET','PUT'])
+def complete_Task():
+    
+    updateTaskListName = request.args.get('taskStatus')
+    listId = get_list_by_name(updateTaskListName)
+
+    taskId = request.args.get('taskId')
+   
+    complete_item(taskId, listId)
+
+    return redirect('/')
+
+
+@app.route('/update/', methods=['GET','PUT'])
+def update_Task():
```

Comment:
This appears to be a duplicate of the above, but with a different name - is there any reason to keep both?